release/packaging/utils/make-packages.sh

```diff
@@ -39,114 +39,108 @@ for package_type in "$@"; do
 
     case ${package_type} in
 
-	deb )
-	    if [ "${PKG_NAME}" = felix ]; then
-	        # Felix is built with RHEL/UBI and links against libpcap.so.1. We need this patchelf
-	        # until Debian changes the soname from .0.8 to .1.
-	        # FIXME remove the following patchelf command once Debian dependency is updated.
-	        patchelf --replace-needed libpcap.so.1 libpcap.so.0.8 bin/calico-felix
-	    fi
-
-	    # The Debian version that we are about to generate.
-	    debver=${FORCE_VERSION_DEB:-`git_version_to_deb ${version}`}
-	    debver=`strip_v ${debver}`
-
-	    # Current time in Debian changelog format; e.g. Wed, 02
-	    # Mar 2016 14:08:51 +0000.
-	    timestamp=`date "+%a, %d %b %Y %H:%M:%S %z"`
-	    for series in focal jammy; do
-		{
-			if ${release}; then
-				changelog_message="* ${NAME} v${version} (from Git commit ${sha})."
-			else
-				changelog_message="* Development snapshot (from Git commit ${sha})."
-			fi
-		    cat <<EOF
-${PKG_NAME} (${DEB_EPOCH}${debver}-$series) $series; urgency=low
-
-  ${changelog_message}
-
- -- Daniel Fox <dan.fox@tigera.io>  ${timestamp}
-EOF
-		} > debian/changelog
-
-		excludes="${DPKG_EXCL:--I}"
-
-		${DOCKER_RUN_RM} calico-build/${series} dpkg-buildpackage ${excludes} -S -d | ts "[build $series]"
-	    done
-
-	    cat <<EOF
+    deb )
+        if [ "${PKG_NAME}" = felix ]; then
+            # Felix is built with RHEL/UBI and links against libpcap.so.1. We need this patchelf
+            # until Debian changes the soname from .0.8 to .1.
+            # FIXME remove the following patchelf command once Debian dependency is updated.
+            patchelf --replace-needed libpcap.so.1 libpcap.so.0.8 bin/calico-felix
+        fi
+
+        # The Debian version that we are about to generate.
+        debver=${FORCE_VERSION_DEB:-$(git_version_to_deb "${version}")}
+        debver=$(strip_v "${debver}")
+
+        # Current time in Debian changelog format; e.g. Wed, 02
+        # Mar 2016 14:08:51 +0000.
+        timestamp=$(date "+%a, %d %b %Y %H:%M:%S %z")
+        for series in focal jammy noble; do
+                if ${release}; then
+                        changelog_message="${NAME} v${debver} (from Git commit ${sha})."
+                else
+                        changelog_message="Development snapshot (from Git commit ${sha})."
+                fi
+                rm -f debian/changelog debian/changelog.dch
+                dch --controlmaint --create --package "${PKG_NAME}" -v "${DEB_EPOCH}${debver}-$series" "${changelog_message}"
+                dch --controlmaint --release ""
```

Comment:
Does this still put you as the author?  If so, I'm curious where that is configured.